utils/fdb_wrapper_cpp/src/fdbcs.cpp

```diff
@@ -19,10 +19,20 @@
 #include <iostream>
 #include <thread>
 #include <chrono>
-#include "../include/fdbcs.hpp"
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_io.hpp>
+#include <boost/uuid/random_generator.hpp>
+#include <boost/lexical_cast.hpp>
+#include <chrono>
+#include "fdbcs.hpp"
 
 namespace FDBCS
 {
+
+#define RETURN_ON_ERROR(retcode) \
```

Comment:
Can we do the same without C PP macro?

Reply:
I'm not sure how to make a function on c++ which return only on specific case (error) and not return in other case. Can you please show an example